go-controller/pkg/ovn/master_test.go

```diff
@@ -853,6 +853,19 @@ subnet=%s
 })
 */
 
+func startFakeController(oc *DefaultNetworkController, wg *sync.WaitGroup) []*net.IPNet {
```

Comment:
Dan, you are the master of spotting duplicate code and cleaning it up. Thank you!